### PR TITLE
TypeScript & cleanup

### PR DESCRIPTION
**Pull request by forshtat**

### CODE REVIEW

File: src/cli/CommandsLogic.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@ import { ether } from '@openzeppelin/test-helpers'
 import { HttpProvider, TransactionReceipt } from 'web3-core'
 import { merge } from 'lodash'
 
-import { sleep } from '../common/utils'
+import { sleep } from '../common/Utils'
```
[the remark this answers is not in the document]
Comment (forshtat) [reply src/cli/CommandsLogic.ts:9]: What do you mean? All files in our repo except for this one were already camel-cased, and it works fine on linux.

File: test/TestBatchForwarder.test.ts [hunk 2]
```diff
@@ -0,0 +1,166 @@
+/* global contract artifacts before it */
```
[the remark this answers is not in the document]
Comment (forshtat) [reply test/TestBatchForwarder.test.ts:1]: This is the last time I promise!

File: test/TestBatchForwarder.test.ts [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+        paymaster: paymaster.address,
+        forwarder: forwarder.address
+      }
+    }
+  })
+
+  context('#sendBatch', function () {
+    it('should send all methods in the batch', async () => {
+      const relayRequest = cloneRelayRequest(sharedRelayRequestData)
+      relayRequest.relayData.senderNonce = (await forwarder.getNonce(from)).toString()
```
[the remark this answers is not in the document]
Comment (forshtat) [reply test/TestBatchForwarder.test.ts:79]: It is not 'nicer', but this is type-safe. JavaScript way was "just mix these two bags of rubbish together", this is "assign new values to variables". 
My regret here is that JS/TS does not have a built-in clone/copy. Seems like a useful thing, you know.

File: src/cli/CommandsLogic.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (drortirosh) [reply src/cli/CommandsLogic.ts:9]: hm.. didn't see a rename, but its lowercase on my machine. but it passes, to its ok.

File: test/TestBatchForwarder.test.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (drortirosh) [reply test/TestBatchForwarder.test.ts:1]: give it a try. the last time was 2 times ago.

File: test/TestBatchForwarder.test.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (drortirosh) [reply test/TestBatchForwarder.test.ts:79]: but we clone and modify all over the tests. so maybe the clone method should be given the override, and merge them:
e.g
```
return { gasData: { ...orig.gasData, override.gasData }, 
   relayData: { ...orig.relayData, override.relayData } }
```

**Pull request merged**